### PR TITLE
feat: add mock SMS sender

### PR DESCRIPTION
**Pull request by kofj**
Add mock sms sender, nothing to do.

Signed-off-by: 疯魔慕薇 <kfanjian@gmail.com>

### CODE REVIEW

**Comment by kofj** [comment]
About https://github.com/casdoor/casdoor/issues/1002#issue-1334439744

File: basic.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ const (
 	VolcEngine   = "Volc Engine SMS"
 	Huyi         = "Huyi SMS"
 	HuaweiCloud  = "Huawei Cloud SMS"
+	MockSMS      = "Mock SMS"
```
Comment (hsluoyz) [inline basic.go]: `MockSMS` -> `MockSms`

File: basic.go [hunk 2]
```diff
@@ -40,6 +41,9 @@ func NewSmsClient(provider string, accessId string, accessKey string, sign strin
 		return GetHuyiClient(accessId, accessKey, template)
 	case HuaweiCloud:
 		return GetHuaweiClient(accessId, accessKey, sign, template, other)
+	case MockSMS:
+		return NewMocker(accessId, accessKey, sign, template, other)
+
```
Comment (hsluoyz) [inline basic.go]: no empty line

File: mock.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package go_sms_sender
```
Comment (hsluoyz) [inline mock.go:15]: apache header

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**